Generate.py

```diff
@@ -432,7 +432,7 @@ def handle_option(ret: argparse.Namespace, game_weights: dict, option_key: str,
             raise Exception(f"Error generating option {option_key} in {ret.game}") from e
         else:
             if hasattr(player_option, "verify"):
-                player_option.verify(AutoWorldRegister.world_types[ret.game], plando_options)
+                player_option.verify(AutoWorldRegister.world_types[ret.game], plando_options, ret.name)
```

Comment:
maybe a stupid idea, but i think it should probably be possible to use our metaclass magic to inject a `def verify(...): pass` into any Option that does not have a Verify mixin* to get rid of the hasattr.

*during class constructions you should have a list of all the classes the class is built from. If you run hasattr("verify") on all of them, and they all come back negative, you can add a "blank" verify to the resulting class.